src/cplus_plugin/definitions/defaults.py

```diff
@@ -23,7 +23,21 @@
 )
 
 OPTIONS_TITLE = "CPLUS"  # Title in the QGIS settings
+REPORT_OPTIONS_TITLE = "Reporting"
+LOG_OPTIONS_TITLE = "Logs"
 ICON_PATH = ":/plugins/cplus_plugin/icon.svg"
+REPORT_SETTINGS_ICON_PATH = str(
```

Comment:
Let's use one of the QGIS layouts icons with a custom CPLUS "green" color for the report settings icon.